web/models/project_skill.ex

```diff
@@ -11,7 +11,7 @@ defmodule CodeCorps.ProjectSkill do
   @doc """
   Builds a changeset based on the `struct` and `params`.
   """
-  def changeset(struct, params \\ %{}) do
+  def create_changeset(struct, params \\ %{}) do
```

Comment:
Thew new `load_and_authorize_changeset` plug automatically calls the changeset method, inferred from action name - `#{action}_changeset`. Due to that, I renamed a couple of these. 

It's a similar approach to what `Canary` uses for `load_resource`.